src/curated-corpus/components/SidebarWrapper/SidebarWrapper.tsx

```diff
@@ -3,8 +3,7 @@ import { DateTime } from 'luxon';
 import { useStyles } from './SidebarWrapper.styles';
 import { ScheduleSummaryConnector } from '../ScheduleSummaryConnector/ScheduleSummaryConnector';
 import { SidebarDatePicker } from '../SidebarDatePicker/SidebarDatePicker';
```

Comment:
Could these imports be cleaned up? Both components should be accessible to import from the index file in the `components` folder, so just from `'../`